tests/test_vpc.rb

```diff
@@ -56,247 +49,251 @@ def test_vpc_create
 
   def test_vpc_destroy
     # create and test again
-    @vpc = Vpc.new(100)
-    @vpc.destroy
+    vpc = Vpc.new(100)
+    vpc.destroy
     refute(Vpc.enabled, 'VPC feature should have been disabled')
```

Comment:
The problem is that we may have other providers that may need a given feature to be enabled; ie. if I did 'no feature pim' to clean up the global config it would also clear out any interface pim configs along with any other pim provider configs.

Yes, 'no feature' does make config cleanup easy but we can do the same thing in this case by calling each setter with it's default value, ie:

```
  def self.destroy
    foo=(default_foo)
    bar=(default_bar)
  end
```

I realize this approach may not work in all cases. Where it does work, we should use it. 
Again, my comment was just an fyi for this review.